services/base-jdbc/src/main/resources/META-INF/native-image/org/eclipse/hono/hono-service-base-jdbc/resources-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "resources": {
+    "includes": [
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/devcon\\/base.h2.sql.yaml" },
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/devcon\\/base.postgresql.sql.yaml" },
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/devcon\\/base.sql.yaml" },
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/device\\/base.h2.sql.yaml" },
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/device\\/base.postgresql.sql.yaml" },
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/device\\/base.sql.yaml" },
+      { "pattern":  "org\\/eclipse\\/hono\\/service\\/base\\/jdbc\\/store\\/tenant\\/base.sql.yaml" },
+      { "pattern":  "sql\\/h2\\/01-create.tenants.sql" },
+      { "pattern":  "sql\\/h2\\/02-create.devices.sql" },
+      { "pattern":  "sql\\/postgresql\\/01-create.tenants.sql" },
+      { "pattern":  "sql\\/postgresql\\/02-create.devices.sql" }
```

Comment:
maybe we can make these more generic, e.g. using wildcards that expand to all `yaml` and `sql` files?

Reply:
Done, I kept some base resource directory structure under which *.yaml and *.sql files are filtered in.